src/main/java/net/minecraftforge/server/console/ConsoleCommandCompleter.java

```diff
@@ -45,10 +46,9 @@ public ConsoleCommandCompleter(DedicatedServer server)
     }
 
     @Override
-    public int complete(String buffer, int cursor, List<CharSequence> candidates)
```

Comment:
I think you should deprecate the old method and keep it around, to avoid breaking anything that might use it. Put a `//TODO remove in 1.13` by the deprecated annotation so we can remove it later.

Reply:
The `ConsoleCommandCompleter` isn't exposed anywhere, it's only given to JLine to use it. We cannot keep this method without also keeping the old version of JLine around. 

The actual "breaking" change here is the upgrade from JLine 2.x to JLine 3.x, which is why the old method is no longer useful. On the other hand, JLine was only added for the console, mods should not use it directly (it isn't present on the client either).

Reply:
This class should probably be made package-private at some point, then.

Reply:
That's a good point. I will change it when I update the PR the next time.

Reply:
Class is package-private now.